spring-batch-docs/asciidoc/readersAndWriters.adoc

```diff
@@ -3002,7 +3036,7 @@ instance of the `ScriptItemProcessor`.
 [[classifierCompositeItemProcessor]]
 ===== ClassifierCompositeItemProcessor
```

Comment:
Again, a decorator and should be with them.

Reply:
Again, moving this section involves moving the section, the anchor, and the link.